README.md

```diff
@@ -15,8 +15,11 @@
 
 ## What is Rook?
 
-Rook is a distributed storage system designed for cloud native applications. It
-exposes file, block, and object storage on top of shared resource pools. Rook has minimal
+Rook is a distributed storage system designed for cloud native applications. It aims to be the storage solution for container-native app developers. 
+Just like your container apps, Rook is designed to take advantage of the resource management of orchestration platform, Kubernetes. 
```

Comment:
Just like your container apps, Rook is designed to take advantage of the resource management and orchestration of the Kubernetes platform.